src/main/java/eladkay/scanner/RegistryEventHandler.java

```diff
@@ -0,0 +1,59 @@
+package eladkay.scanner;
+
+import eladkay.scanner.biome.TileEntityBiomeScanner;
+import eladkay.scanner.init.ModBlocks;
+import eladkay.scanner.terrain.TileEntityScannerQueue;
+import eladkay.scanner.terrain.TileEntityTerrainScanner;
+import net.minecraft.block.Block;
+import net.minecraft.client.renderer.block.model.ModelResourceLocation;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemBlock;
+import net.minecraftforge.client.event.ModelRegistryEvent;
+import net.minecraftforge.client.model.ModelLoader;
+import net.minecraftforge.event.RegistryEvent;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
+import net.minecraftforge.fml.common.registry.GameRegistry;
+
+import static eladkay.scanner.ScannerMod.logger;
+
+
+@Mod.EventBusSubscriber
+public class RegistryEventHandler {
```

Comment:
The original version used a library mod called LibrarianLib. It was really good and while I would understand if you rather stay with the vanilla way here (really! no pressure!), I'm obligated to plug in @Demoniaque to have him lecture about you about how insanely  good it was.